bundle/tests/scorecard/kuttl/test-multi-db-single/01-pods.yaml

```
@@ -0,0 +1,81 @@
+---
+apiVersion: cloud.redhat.com/v1alpha1
+kind: ClowdEnvironment
+metadata:
+  name: test-multi-db-single
+spec:
+  targetNamespace: test-multi-db-single
+  providers:
+    web:
+      port: 8000
+      mode: operator
+    metrics:
+      port: 9000
+      mode: operator
+      path: "/metrics"
+    kafka:
+      mode: none
+    db:
+      mode: single
```

Looks like the test needs to be 
```suggestion
      mode: shared
```